twitchio/ext/routines/__init__.py

```diff
@@ -330,6 +330,22 @@ def start_time(self) -> Optional[datetime.datetime]:
         """
         return self._start_time
 
+    @property
+    def next_run(self) -> Optional[datetime.datetime]:
```

Comment:
Not sure why it's `optional`? It doesn't appear the return would ever be optional